pkg/storaged/stratis-details.jsx

```diff
@@ -207,6 +216,153 @@ export const StratisPoolDetails = ({ client, pool }) => {
         });
     }
 
+    function add_passphrase() {
+        dialog_open({
+            Title: _("Add passphrase"),
+            Fields: [
+                PassInput("passphrase", _("Passphrase"),
+                          { validate: val => !val.length && _("Passphrase cannot be empty") }),
+                PassInput("passphrase2", _("Confirm"),
+                          { validate: (val, vals) => vals.passphrase.length && vals.passphrase != val && _("Passphrases do not match") })
+            ],
+            Action: {
+                Title: _("Add"),
+                action: vals => {
+                    return get_unused_keydesc(client, pool.Name)
+                            .then(keydesc => {
+                                return with_stored_passphrase(client, keydesc, vals.passphrase,
+                                                              () => pool.BindKeyring(keydesc))
+                                        .then(std_reply);
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230601-113309-0e7aeb29-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)